messages/recordcounts.json

```diff
@@ -0,0 +1,8 @@
+{
+  "commandDescription": "print record counts for the given objects",
```

Comment:
Small edit (just to keep the description similar to other display commands):  "display record counts for the specified standard and custom objects"

Note: I assume this command works on both standard and custom objects.  If it works on only standard objects, then edit the description as needed.

Reply:
Thank you for your review. I've incorporated your suggested change.